themes/SuiteP/include/ListView/ListViewGeneric.tpl

```diff
@@ -317,3 +324,15 @@ function lvg_nav(m,id,act,offset,t){
 </script>
 <script type="text/javascript" src="include/InlineEditing/inlineEditing.js"></script>
 {/if}
+
+
+{if isset($templateMeta)}
```

Comment:
Ok, understand that - but should it be consistent with other areas of the system.  For example EditView includes 'includes' via the EditView2.php

$this->th->ss->assign('includes', isset($this->defs['templateMeta']['includes']) ? $this->defs['templateMeta']['includes'] : null);